workers/social/lib/social/models/group/validators.coffee

```diff
@@ -3,7 +3,7 @@
 getRoles = (permission, permissionSet) ->
   return []  unless permissionSet
   roles = ( perm.role for perm in permissionSet.permissions \
-                      when permission in perm.permissions   )
+                      when permission in perm?.permissions   )
```

Comment:
```coffee
when in (perm?.permissions or [])
```

might be safer, just in case. (not sure if it throws or not when `foo in undefined` happens)